src/pages/blog/[slug].js

```diff
@@ -99,6 +104,12 @@ export const getStaticProps = async context => {
 
   const readingTime = getReadingTime(blog?.content, locale)
 
+  const {
+    author: { id },
+  } = blog
+
+  const blogs = await getAuthorBlogs(locale, id)
```

Comment:
No need to destructor the id as we don't use it multiple times.

```suggestion
  const blogs = await getAuthorBlogs(locale, blog.author.id, blog.id)
```